features/environment.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=W0601, W0613
+from behave import use_fixture
+
+from features.fixtures import selenium_browser
+
+
+def setup_debug_on_error(userdata):
+    global BEHAVE_DEBUG_ON_ERROR
+    BEHAVE_DEBUG_ON_ERROR = userdata.getbool("BEHAVE_DEBUG_ON_ERROR")
```

Comment:
maybe this can be part of Django `testting` settings? @rwrzesien, what do you think?

Reply:
Done, it got removed entirely

Reply:
Depending on if we want to run this in Django context. If yes, then it is OK, but I thought originally that we want to run it independently. Do we?